lib/src/wiggle_abi/dictionary_impl.rs

```diff
@@ -9,20 +12,63 @@ use {
     wiggle::GuestPtr,
 };
 
+use memoize::memoize;
+
+#[memoize]
+fn read_json_file(file: String) -> serde_json::Map<String, serde_json::Value> {
+    let data = fs::read_to_string(file).expect("Unable to read file");
+    let json: serde_json::Value = serde_json::from_str(&data).expect("JSON was not well-formatted");
```

Comment:
We would need to document somewhere what the expected schema is for the JSON file. Perhaps on [the docs site](https://developer.fastly.com/reference/fastly-toml/#local-server)?

The schema currently is `Object<String, String>`

Reply:
If you want to jam some arbitrary bytes into a JSON string, typically you'd base64-encode them. Will serde_json handle that properly, i.e. give you the actual bytes back rather than their base64 encoding?

Reply:
Ah, sorry! 😅 I'm not asking for a change or anything, I'm just asking about the present behavior. But I guess Edge Dictionaries currently model values as strings rather than arbitrary bytes, so there's no reason to do anything differently here. Sorry for the distraction!